HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1487,10 +1517,22 @@ def min_value_zero
     column_names = SchedulesFile.ColumnNames
     column_names.each do |column_name|
       min_value_zero[column_name] = true
-      if SchedulesFile.SetpointColumnNames.include? column_name
+      if SchedulesFile.SetpointColumnNames.include?(column_name)
         min_value_zero[column_name] = false
       end
     end
     return min_value_zero
   end
+
+  def only_zeros_and_ones
+    only_zeros_and_ones = {}
+    column_names = SchedulesFile.ColumnNames
+    column_names.each do |column_name|
+      only_zeros_and_ones[column_name] = false
+      if SchedulesFile.OperatingModeColumnNames.include?(column_name)
```

Comment:
Should the `vacancy` column be included here? The documentation says only 0s or 1s are allowed for it too.